cdap-ui/app/cdap/components/PipelineConfigurations/ConfigurationsContent/RuntimeArgsTabContent/RuntimeArgsTabContent.scss

```diff
@@ -1,81 +0,0 @@
-/*
- * Copyright © 2018 Cask Data, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
- * use this file except in compliance with the License. You may obtain a copy of
- * the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations under
- * the License.
-*/
-
-#runtime-arguments-tab-content {
-  .step-content-heading {
-    .step-content-subtitle {
-      font-weight: initial;
-      font-size: 12px;
-    }
-  }
-  .runtime-arguments-labels {
-    .key-label {
-      width: 34%;
-
-      &.wider {
-        width: 44%;
-      }
-    }
-
-    .provided-label {
-      width: initial;
-    }
-
-    .provided-popover {
```

Comment:
I think we still need some styling for the provided popover, otherwise it looks broken when you're using macros in Preview